box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxArray.java

```diff
@@ -0,0 +1,43 @@
+package com.box.androidsdk.content.models;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * A collection that contains BoxJsonObject items
+ *
+ * @param <E> the type of elements in this partial collection.
+ */
+public class BoxArray<E extends BoxJsonObject> extends BoxList<E> {
+
+    public static final String PUT_ARRAY = "put_array";
```

Comment:
Let's call this something more generic like "ARRAY" or "ITEMS" as we potentially may use this class for things other than put requests